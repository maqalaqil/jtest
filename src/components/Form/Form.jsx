import React from 'react'

export default function Form() {
    return (
        <div>
            <form>
                <input type="text" />
                <input type="password" />
                <input type="text" />
            </form>
        </div>
    )
}
