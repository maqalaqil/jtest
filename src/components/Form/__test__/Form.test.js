import * as React from "react";
import * as renderer from "react-test-renderer";
import Form from '../Form'

it("this is working head", () => {
  const tree = renderer.create(<Form />).toJSON();
  expect(tree).toMatchSnapshot();
});