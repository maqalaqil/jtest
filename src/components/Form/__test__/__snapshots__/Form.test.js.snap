// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this is working head 1`] = `
<div>
  <form>
    <input
      type="text"
    />
    <input
      type="password"
    />
    <input
      type="text"
    />
  </form>
</div>
`;
